### PR TITLE
assigment 1  : basic http server 

### PR DESCRIPTION
**Pull request by roelof1967**

### CODE REVIEW

**Comment by roelof1967** [comment]
I apolize for this. I thought it would be merged into my earlier push.

**Comment by hueniverse** [comment]
@roelof1967 pull requests are free.

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+var server = new Hapi.Server(); 
+
+server.connection({
+   port: 8000  
+ }); 
+  
+ server.route({
+     method: 'GET', 
+     path: '/version', 
+     handler: function (request, reply) {
```
Comment (hueniverse) [inline lib/index.js]: Missing new line.

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+         reply({ version: file.version })
```
Comment (hueniverse) [inline lib/index.js]: better to do `return reply(...)`

File: lib/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  
+ server.route({
+     method: 'GET', 
+     path: '/version', 
+     handler: function (request, reply) {
+         reply({ version: file.version }) 
+    } 
+ }); 
+  
+server.start(function () {
```
Comment (hueniverse) [inline lib/index.js]: Missing new line. Missing `err` in callback, and checking for errors.

File: lib/index.js (hunk 1 above)
Comment (roelof1967) [inline lib/index.js]: Oke, can you give me a tip where I have missed it. I do not see it.

File: lib/index.js (hunk 3 above)
Comment (roelof1967) [inline lib/index.js]: oke, where did I miss a new line. The callback function I will implement.

File: lib/index.js (hunk 1 above)
Comment (idanwe) [inline lib/index.js]: "Newline after { except for inlined or empty objects"
https://github.com/hapijs/contrib/blob/master/Style.md#newlines

**Pull request closed without merging**